neqo-transport/src/events.rs

```diff
@@ -181,21 +189,23 @@ impl ConnectionEvents {
         }
         if let Some(r) = remove {
             q.remove(r);
-            q.push_back(ConnectionEvent::DatagramLost);
+            q.push_back(ConnectionEvent::IncomingDatagramDropped);
         }
     }
 
     pub fn add_datagram(&self, max_queued_datagrams: usize, data: &[u8]) {
```

Comment:
Why is the number of enqueued datagrams not a property of `self`?  (Something to consider when moving events to their own queue.)

Reply:
I was thinking about moving incoming datagram queue into QuicDatagrams structure. That will not have this issue. We can discuss that in issue #1218